uaa/src/test/java/org/cloudfoundry/identity/uaa/integration/util/IntegrationTestUtils.java

```diff
@@ -86,10 +86,10 @@
 
 public class IntegrationTestUtils {
 
-    public static final String SIMPLESAMLPHP_UAA_ACCEPTANCE = "http://simplesamlphp.uaa-acceptance.cf-app.com";
+    public static final String SIMPLESAMLPHP_UAA_ACCEPTANCE = "http://simplesamlphp2.uaa-acceptance.cf-app.com";
```

Comment:
Not to break the tests when we merge and the tests run on CI, we temporarily changed the URL to point towards the already deployed v2 saml server. We'll revert this after we have deployed the new v2 server to the old URL.